smart-contracts/cookbook/smart-contracts/deploy-basic/remix.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy a Basic Contract with Remix IDE
+description: Learn how to deploy a basic smart contract to Polkadot Hub using Remix IDE Ideal for rapid prototyping, learning, and visual development.
+categories: Smart Contracts
+---
+
+# Deploy a Basic Contract with Remix IDE
+
+This guide demonstrates how to deploy a basic Solidity smart contract to Polkadot Hub using [Remix IDE](https://remix.live/){target=\_blank}, which offers a visual, browser-based environment perfect for rapid prototyping and learning. It requires no local installation and provides an intuitive interface for contract development.
+
+## Prerequisites:
+
+- Basic understanding of Solidity programming.
+- Test tokens for gas fees (available from the [Polkadot faucet](https://faucet.polkadot.io/){target=\_blank}). See the [step-by-step instructions](/smart-contracts/faucet/#get-test-tokens){target=\_blank}.
+- A wallet with a private key for signing transactions.
+
+## Access Remix
+
+Navigate to [https://remix.polkadot.io/](https://remix.polkadot.io/){target=\_blank} in your web browser.
+
+The interface will load with a default workspace containing sample contracts. In this interface, you can access a file explorer, edit your code, interact with various plugins for development, and use a terminal. By default, you will see the `contracts` folder with the `Storage.sol` file:
+
+![](/images/smart-contracts/cookbook/smart-contracts/deploy-basic/deploy-basic-pvm/deploy-basic-pvm-01.webp)
+
+## Compile
+
+1. To compile your contract:
```

Comment:
Ok, so in the previous section we mention "You'll see the `contracts` folder with the `Storage.sol` file. Then here we say "To compile your contract". We never tell them that they're going to be deploying the `Storage.sol` file and that's the contract we're going to use throughout the rest of the example. So we should make that clear